JenkinsfileRT

```diff
@@ -13,7 +13,7 @@ numpy_version = '1.16'
 env_vars = [
     "TEST_BIGDATA=https://bytesalad.stsci.edu/artifactory",
     "CRDS_SERVER_URL=https://jwst-crds.stsci.edu",
-    "CRDS_CONTEXT=jwst_0508.pmap",
+    "CRDS_CONTEXT=jwst_0513.pmap",
```

Comment:
And this can be set to `jwst-edit`

Reply:
I.e. revert this:

https://github.com/spacetelescope/jwst/commit/78fc038294c4e5a2ca5e507f04ca2cb6797ab31a#diff-d989dbe11b6a131293f54f52a6458757

Reply:
Done